src/test/java/TypewiseAlert/TypewiseAlertTest.java

```diff
@@ -10,6 +12,21 @@
     public void infersBreachAsPerLimits()
     {
       assertTrue(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+        BreachType.TOO_LOW);
+    }
+    
+    @Test
```

Comment:
test cases are incomplete !....

add test case for email and controller notification